sdks/java/core/src/main/java/org/apache/beam/sdk/util/state/TimerCallback.java

```diff
@@ -25,4 +25,11 @@
 public interface TimerCallback {
   /** Processes the {@link TimerInternals.TimerData TimerData}. */
   void onTimer(TimerInternals.TimerData timer) throws Exception;
+
+  TimerCallback NO_OP = new TimerCallback() {
```

Comment:
This isn't quite what I was expecting.

Can we enumerate the cases in which `TimerCallback.NO_OP` is used? It would be useful to ensure that we're not discarding any timers (and letting data get stuck as a result)